components/authz-service/storage/v2/postgres/postgres.go

```diff
@@ -498,20 +571,7 @@ func (p *pg) insertOrReusePolicyMemberWithQuerier(ctx context.Context, policyID
 	return err
 }
 
-func (p *pg) getPolicyMembers(ctx context.Context, id string) ([]v2.Member, error) {
-	return p.getPolicyMembersWithQuerier(ctx, id, p.db)
-}
-
 func (p *pg) getPolicyMembersWithQuerier(ctx context.Context, id string, q Querier) ([]v2.Member, error) {
-	var exists bool
-	err := q.QueryRowContext(ctx, `SELECT EXISTS(SELECT 1 FROM iam_policies WHERE id=$1)`, id).Scan(&exists)
-	if err != nil {
-		return nil, errors.Wrap(err, "check policy existence")
-	}
-	if !exists {
-		return nil, storage_errors.ErrNotFound
-	}
```

Comment:
All functions now calling this check for policy's existence as part of project filtering now.